talentsearch/resources/js/stories/Pages.stories.tsx

```diff
@@ -12,3 +13,7 @@ stories.add("Home Page", () => {
 stories.add("Search Page", () => {
   return <SearchPage />;
 });
+
+stories.add("Request Page", () => {
+  return <RequestPage />;
```

Comment:
How can we see this page in storybook if it's behind an auth wall?

Reply:
Yea that's true, do you have something in mind?

Reply:
Yea that's true. I guess I'll remove it then.

Reply:
I think you forgot to remove this.

Reply:
😕